intel_owl/settings/cache.py

```diff
@@ -1,15 +1,6 @@
-import os
-
-from ._util import set_permissions
-
-MEDIA_ROOT = "/opt/deploy/files_required"
```

Comment:
MEDIA_ROOT is used in another part of the application so I'll keep it there for now